### DISCUSSION TITLE
来看一下都是谁要润？

### DISCUSSION THREAD

**Discussion by kumaleap** [The-Run-Philosophy-Organization/run#224] [General]

**Comment by kumaleap** [comment]
中国这么好

**Comment by skyabcher** [comment]
> 中国这么好

如果你在上海，你还会这么说吗

**Comment by kumaleap** [comment]
在上海……

**Reply by Lamerrrrr** [reply]
![image](https://user-images.githubusercontent.com/104127459/164579123-d4da3b62-d540-497b-ab25-8034c1f84a8a.jpeg)

**Comment by skyabcher** [comment]
> 在上海……

还需要抢菜吗

**Comment by the-run-philosophy** [comment]
据我所知，阿里巴巴，腾讯，达摩院都在大规模裁员。你说的好到底是啥意思？好从哪里来？
